build/version.txt

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
This always points to the next version to be released. After every release, minor version is updated here and merged into dev branch (there will be automation for this).

Reply:
Version should be coming from `.csproj` similar to other extensions and webjobs sdk: https://github.com/Azure/azure-webjobs-sdk/pull/2610/files